### PR TITLE
[12.0][FIX] stock_no_negative: Use product display_name instead of name

### PR DESCRIPTION
**Pull request by Abranes**
Use `product.display_name` instead of `product.name`

### CODE REVIEW

File: stock_no_negative/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 {
     'name': 'Stock Disallow Negative',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.0.1',
```
Comment (pedrobaeza) [inline stock_no_negative/__manifest__.py]: Don't change the module version. Now this is done by the bot.

**Review by pedrobaeza — APPROVED** [review]
The rest seems OK, thanks.

File: stock_no_negative/__manifest__.py (hunk 1 above)
Comment (Abranes) [reply stock_no_negative/__manifest__.py]: @pedrobaeza Ok!

**Review by fuentes73 — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
LGTM 👍

**Pull request merged**